apps/hubble-stats/data/getOverviewChipsData.ts

```diff
@@ -1,14 +1,59 @@
-import { randNumber } from '@ngneat/falso';
+import { formatEther } from 'viem';
+import vAnchorClient from '@webb-tools/vanchor-client';
+
+import { vAnchorAddresses, allSubgraphUrls } from '../constants';
 
 type OverviewChipsDataType = {
-  tvl: number;
-  volume: number;
+  tvl: number | undefined;
+  volume: number | undefined;
 };
 
 export default async function getOverviewChipsData(): Promise<OverviewChipsDataType> {
-  await new Promise((r) => setTimeout(r, 1000));
+  let tvl, volume;
+
+  try {
+    const tvlVAnchorsByChainsData =
+      await vAnchorClient.TotalValueLocked.GetVAnchorsTotalValueLockedByChains(
+        allSubgraphUrls,
+        vAnchorAddresses
+      );
+
+    tvl = tvlVAnchorsByChainsData.reduce((tvlTotal, vAnchorsByChain) => {
```

Comment:
The `tvlVAnchorsByChainsData` would be `null` here if `allSubgraphUrls` or `vAnchorAddresses` are invalid. Therefore, this could throw an error here.

Reply:
There are 2 cases here:
- There were errors when querying -> Promise is rejected -> tvl will be `undefined`
- The queries run fine but with either or both `allSubgraphUrls` and `vAnchorAddresses` are invalid -> Promise is resolved but returns an empty array -> tvl is equal to 0 (I believe this is the correct behavior) 

Let me know your thoughts on this!

Reply:
Sounds good!